_health-care/_js/components/form-elements/ErrorableCheckbox.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import _ from 'lodash';
+
+/**
+ * A form checkbox with a label that can display error messages.
+ *
+ * Validation has the following props.
+ * `errorMessage` - Error string to display in the component.
+ *                  When defined, indicates checkbox has a validation error.
+ * `checkedValue` - String. The value that should be saved when checked.
+ * `label` - String for the checkbox label.
+ * `required` - boolean. Render marker indicating field is required.
+ * `value` - string. Value of the checkbox.
+ * `onValueChange` - a function with this prototype: (newValue)
+ */
+class ErrorableCheckbox extends React.Component {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.inputId = _.uniqueId('errorable-checkbox-');
+  }
+
+  handleChange() {
+    let newValue = '';
+
+    if (this.props.value === '') {
+      newValue = this.props.checkedValue;
+    }
+
+    this.props.onValueChange(newValue);
+  }
+
+  render() {
+    // Calculate error state.
+    let errorSpan = '';
```

Comment:
Hmmm... soon we should extract this into a utility function. Save it for another PR though.